info.colobot.Colobot.yaml

```diff
@@ -0,0 +1,179 @@
+---
+app-id: info.colobot.Colobot
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: colobot
+rename-icon: colobot
+rename-desktop-file: colobot.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--env=LD_LIBRARY_PATH=/app/lib64:/app/lib"
+build-options:
+  cflags: "-O2 -g"
+  cxxflags: "-O2 -g"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/share/doc"
+  - "/man"
+  - "/share/man"
+  - "/share/gtk-doc"
+  - "/share/vala"
+  - "*.la"
+  - "*.a"
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - "./bootstrap.sh --prefix=/app --with-libraries=system,filesystem,regex"
+      - "./b2 headers"
+      - "./b2 install"
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2
+        sha256: 7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+
+  - name: physfs
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPHYSFS_BUILD_TEST=off
+    sources:
+      - type: archive
+        url: https://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2
+        sha256: b77b9f853168d9636a44f75fca372b363106f52d789d18a2f776397bf117f2f1
+
+  # Provided libsndfile is compiled without ogg/vorbis support
```

Comment:
Report this against the runtime: https://gitlab.com/freedesktop-sdk/freedesktop-sdk

Both libogg and libvorbis are there so it can be enabled probably.

Reply:
I've opened up [an issue](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/406) there, I guess depending on the response this PR might stall until sndfile is fixed in the runtime.

Reply:
fdo folks are pretty quick, shouldn't be a problem =)